polaris-react/src/components/Button/Button.scss

```diff
@@ -351,7 +344,6 @@
     &:active {
       // stylelint-disable-next-line polaris/conventions/polaris/custom-property-allowed-list -- Button custom property
       --pc-button-text: var(--p-color-text-inverse-subdued);
-      box-shadow: var(--p-shadow-button-inset-experimental);
```

Comment:
Shadow for critical and success primary button active states were combined so I've removed them here to add to the individual classes instead with the token replacement value.